website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,387 @@
+To get root shell access to a node, execute the aforementioned script by supplying the target node name as an argument:
+
+```
+$ <path-to-ops-toolbelt-repo>/hacks/ops-pod <target-node>
+```
+
+## Custom root pod
+
+Alternatively, a pod can be [assigned](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/) to a target node and a shell can 
+be opened via [standard Kubernetes means](https://kubernetes.io/docs/tasks/debug-application-cluster/get-shell-running-container/). 
+To enable root access to the node, the pod specification requires proper `securityContext` and `volume` properties.
+
+For instance you can use the following pod manifest, after changing <target-node-name> with the name of the node you want this pod attached to:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: privileged-pod
+  namespace: default
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: <target-node-name>
+  containers:
+  - name: busybox
+    image: busybox
+    stdin: true
+    securityContext:
+      privileged: true
+    volumeMounts:
+    - name: host-root-volume
+      mountPath: /host
+      readOnly: true
+  volumes:
+  - name: host-root-volume
+    hostPath:
+      path: /
+  hostNetwork: true
+  hostPID: true
+  restartPolicy: Never
+```
+
+# SSH access to a node that failed to join the cluster
+
+This section explores two options that can be used to get SSH access to a node that failed to join the cluster.
+As it is not possible to schedule a pod on the node, the Kubernetes-based methods explored so far cannot be used in this scenario.
+
+Additionally, Gardener typically provisions worker instances in a private subnet of the VPC, hence - there is no public IP address that could be used for direct SSH access.
+
+For this scenario, cloud providers typically have extensive documentation (e.g [AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstances.html) & [GCP](https://cloud.google.com/compute/docs/instances/connecting-to-instance) 
+and in [some cases tooling support](https://cloud.google.com/compute/docs/instances/connecting-advanced#vpn)). 
+However, these approaches are mostly CloudProvider specific, require interaction via their CLI and API or sometimes 
+the installation of a [cloud provider specific agent](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-install-ssm-agent.html) one the node.
+
+Alternatively, `gardenctl` can be used providing a cloud provider agnostic and out-of-the-box support to get ssh access to an instance in a private subnet. 
+Currently `gardenctl` supports AWS, GCP, Openstack, Azure and Alibaba Cloud.
+
+## Identifying the problematic instance
+First, the problematic instance has to be identified. 
+In Gardener, worker pools can be created in different cloud provider regions, zones and accounts. 
+Identifying the problematic instance is the first stepping-stone. 
```

Comment:
The first and last sentences in this paragraph sound redundant. My proposal is to remove the first.

```suggestion
In Gardener, worker pools can be created in different cloud provider regions, zones and accounts. 
Identifying the problematic instance is the first stepping-stone. 
```